Showcase/Showcase/Pages/Tokens/Color/NamedColor/NamedColor+Decorative.swift

```diff
@@ -0,0 +1,136 @@
+            case .colorDecorativeNeutralMuted:
+                return theme.colorDecorativeNeutralMuted
+            case .colorDecorativeNeutralDefault:
+                return theme.colorDecorativeNeutralDefault
+            case .colorDecorativeNeutralEmphasized:
+                return theme.colorDecorativeNeutralEmphasized
+            case .colorDecorativeAccent1Muted:
+                return theme.colorDecorativeAccent1Muted
+            case .colorDecorativeAccent1Default:
+                return theme.colorDecorativeAccent1Default
+            case .colorDecorativeAccent1Emphasized:
+                return theme.colorDecorativeAccent1Emphasized
+            case .colorDecorativeAccent2Muted:
+                return theme.colorDecorativeAccent2Muted
+            case .colorDecorativeAccent2Default:
+                return theme.colorDecorativeAccent2Default
+            case .colorDecorativeAccent2Emphasized:
+                return theme.colorDecorativeAccent2Emphasized
+            case .colorDecorativeAccent3Muted:
+                return theme.colorDecorativeAccent3Muted
+            case .colorDecorativeAccent3Default:
+                return theme.colorDecorativeAccent3Default
+            case .colorDecorativeAccent3Emphasized:
+                return theme.colorDecorativeAccent3Emphasized
+            case .colorDecorativeAccent4Muted:
+                return theme.colorDecorativeAccent4Muted
+            case .colorDecorativeAccent4Default:
+                return theme.colorDecorativeAccent4Default
+            case .colorDecorativeAccent4Emphasized:
+                return theme.colorDecorativeAccent4Emphasized
+            case .colorDecorativeAccent5Muted:
+                return theme.colorDecorativeAccent5Muted
+            case .colorDecorativeAccent5Default:
+                return theme.colorDecorativeAccent5Default
+            case .colorDecorativeAccent5Emphasized:
+                return theme.colorDecorativeAccent5Emphasized
+            case .colorDecorativeSkintTint100:
+                return theme.colorDecorativeSkintTint100
+            case .colorDecorativeSkintTint200:
+                return theme.colorDecorativeSkintTint200
+            case .colorDecorativeSkintTint300:
+                return theme.colorDecorativeSkintTint300
+            case .colorDecorativeSkintTint400:
+                return theme.colorDecorativeSkintTint400
+            case .colorDecorativeSkintTint500:
+                return theme.colorDecorativeSkintTint500
+            case .colorDecorativeSkintTint600:
+                return theme.colorDecorativeSkintTint600
+            case .colorDecorativeSkintTint700:
+                return theme.colorDecorativeSkintTint700
+            case .colorDecorativeSkintTint800:
+                return theme.colorDecorativeSkintTint800
+            case .colorDecorativeSkintTint900:
+                return theme.colorDecorativeSkintTint900
+            }
+        }
+    }
+}
```

Comment:
Normalement tu dois avoir des warnings tels que :
- "Blanket Disable Command Violation: The disabled 'function_body_length' rule should be re-enabled before the end of the file (blanket_disable_command)"

Les warnings désactivés dans un fichier doivent être réactivés ensuite.

Il faut ajouter à la fin du fichier, après un saut de ligne, les lignes : 

```swift
// swiftlint:enable function_body_length
```